serverless/guest/context_llm.go

```diff
@@ -8,6 +10,6 @@ func (c *GuestContext) WriteLLMResult(result string) error {
 	panic("not implemented")
 }
 
-func (c *GuestContext) ReadLLMFunctionCall(fnCall any) error {
+func (c *GuestContext) ReadLLMFunctionCall() (*ai.FunctionCall, error) {
```

Comment:
Is there a better name?

Reply:
Do you have any suggestion, I think this variable name is not bad.

Reply:
I suggest remove `Read`, like this: `LLMFunctionCall()`